Update sql_segments.md (#9612)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/segments/sql_segments.md

## Creating SQL Segment Extensions

> To run SQL segment queries, a user must have PII permissions, since it is possible to access PII data via this feature.

Reviewer: ```suggestion
To run SQL segment queries, a user must have PII permissions because it's possible to access PII data through this feature.
```
